### DISCUSSION TITLE
What is the license?

### DISCUSSION THREAD

**Discussion by softhorizons** [joonaspaakko/ScriptUI-Dialog-Builder-Joonas#106] [Q&A]
Can't seem to find one anywhere. It's clear from context you want to share this, but my org requires an official statement of license. MIT, BSD, ...?

**Comment by joonaspaakko** [comment]
No license. Going by [what Github says about it](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository#choosing-the-right-license):

> You're under no obligation to choose a license. However, without a license, the default copyright laws apply, meaning that you retain all rights to your source code and no one may reproduce, distribute, or create derivative works from your work.

You can use  the app in a commercial setting as is. No attribution needed. Any exported code is yours and you can do whatever you want with it.

----

> This has nothing to do with license, but I like to inform commercial users that the repository can be downloaded and used locally by opening `/build/index.html` in your browser, Which might be preferred for preservation.
